test_utilities/bin/dart_test_runner.sh

```diff
@@ -14,10 +14,11 @@ pushd "$1" > /dev/null
 flutter clean
 dart pub get
 
-echo "############# update proto generated code #############"
-apt-get install protocol-compiler
-protoc --dart_out=. app_dart/lib/src/model/proto/**/*.proto
-echo "#######################################################"
+# TODO(drewroen): Validate proto code has been generated.
```

Comment:
https://dart-lang.github.io/linter/lints/flutter_style_todos.html

Reply:
Fixed! Feel free to resolve.

Reply:
It's still missing. Format should be:

TODO(username): Short description. https://github/issue/link

Reply:
Sorry about that. I thought the issue was with the incorrect username. I've added an additional issue link.